test/unit/calculators/student_finance_calculator_test.rb

```diff
@@ -39,6 +39,70 @@ class StudentFinanceCalculatorTest < ActiveSupport::TestCase
         assert_equal "uk-full-time", calculator.course_type
       end
 
+      context "#eligible_for_childcare_grant_one_child?" do
+        should "have low household income and no dependencies" do
+          calculator = StudentFinanceCalculator.new(
+            household_income: 18_441,
+          )
+          assert_not calculator.eligible_for_childcare_grant_one_child?
+        end
+
+        should "have high household income and no dependencies" do
+          calculator = StudentFinanceCalculator.new(
+            household_income: 18_442,
+          )
+          assert_not calculator.eligible_for_childcare_grant_one_child?
+        end
+
+        should "have low household income and adult dependant" do
+          calculator = StudentFinanceCalculator.new(
+            household_income: 18_441,
+            uk_ft_circumstances: %w(children-under-17),
+          )
+          assert calculator.eligible_for_childcare_grant_one_child?
+        end
+
+        should "have high household income and adult dependant" do
+          calculator = StudentFinanceCalculator.new(
+            household_income: 18_442,
+            uk_ft_circumstances: %w(children-under-17),
+          )
+          assert_not calculator.eligible_for_childcare_grant_one_child?
```

Comment:
I feel like `refute` would be an easier read instead of `assert_not`?

Reply:
Unfortunately [the linter _refutes_ this](https://www.rubydoc.info/github/rubocop-hq/rubocop/master/RuboCop/Cop/Rails/RefuteMethods).

Reply:
😱